andela/zinnia-ah-frontend
# #165739510 Setup semantic ui and implement Authentication templates

#### What does this PR do?
 Setup semantic ui and implement Authentication templates

#### Description of Task to be completed?
- [x] install semantic-ui-css & semantic-ui-react

#### How should this be manually tested?
* Clone the repo,
* Run npm install to install all dependencies,
* Run npm start
You should see the images below when you navigate to `/auth`, `/forgot-password`, `reset-password`

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?(if applicable)
#165739510

#### Screenshots
<img width="1440" alt="Screenshot 2019-05-01 at 7 08 35 PM" src="https://user-images.githubusercontent.com/31692932/57069211-e35a0800-6ccb-11e9-9375-210fd299bb1b.png">
<img width="1440" alt="Screenshot 2019-05-01 at 7 09 21 PM" src="https://user-images.githubusercontent.com/31692932/57069212-e3f29e80-6ccb-11e9-85b6-f01874d1ac07.png">
<img width="1440" alt="Screenshot 2019-05-02 at 12 53 21 AM" src="https://user-images.githubusercontent.com/31692932/57069213-e3f29e80-6ccb-11e9-905a-3c774cd87ae1.png">
<img width="1440" alt="Screenshot 2019-05-02 at 12 52 07 AM" src="https://user-images.githubusercontent.com/31692932/57069214-e48b3500-6ccb-11e9-8be2-2528c0dccc99.png">
